src/dso_api/settings.py

```diff
@@ -231,17 +236,23 @@
     LOGGING["handlers"]["azure"] = {
         "level": "DEBUG",
         "class": "opencensus.ext.azure.log_exporter.AzureLogHandler",
-        "connection_string": connection_string,
+        "connection_string": AZURE_APPI_CONNECTION_STRING,
         "formatter": "azure",
     }
-    LOGGING["handlers"]["audit_azure"] = LOGGING["handlers"]["azure"].copy()
+    LOGGING["handlers"]["audit_azure"] = {
+        "level": "DEBUG",
+        "class": "opencensus.ext.azure.log_exporter.AzureLogHandler",
+        "connection_string": AZURE_APPI_AUDIT_CONNECTION_STRING,
+        "formatter": "azure",
+    }
+
     LOGGING["handlers"]["audit_azure"]["formatter"] = "audit_azure"
```

Comment:
Oh, here it is corrected. Best to use one. Set it correct immediately in line 246 and ditch 249. Or keep 249 but then don't set 246